### PR TITLE
[blocky] metrics refactor

### PR DESCRIPTION
**Pull request by jr0dd**
Multiple changes here. Version bump, <s>add configmap boolean</s>, but more importantly refactor metrics section add prom rules. Enabling metrics simply enables the servicemonitor in this chart. @onedr0p I just started with Blocky for the prom rules addition based on our discussion.

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Comment by onedr0p** [comment]
Initial impression is good, I will need to take a deeper look this afternoon or tomorrow.

**Comment by jr0dd** [comment]
> Initial impression is good, I will need to take a deeper look this afternoon or tomorrow.

No rush at all. Just using this one as a base for the rest.

File: charts/stable/blocky/README.md [hunk 1]
```diff
@@ -75,16 +75,22 @@ N/A
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| config | string | see URL to default config | Full list of options https://github.com/0xERR0R/blocky/blob/master/docs/config.yml |
+| configmap | object | see URL to default config | Full list of options https://github.com/0xERR0R/blocky/blob/master/docs/config.yml |
+| configmap.enabled | bool | `false` | If set to 'true', the configuration will be read from these values. |
```
Comment (onedr0p) [inline charts/stable/blocky/README.md:79]: I think you need to re0run helm docs

Comment (jr0dd) [reply charts/stable/blocky/README.md]: done

**Review by onedr0p — APPROVED** [review]

**Pull request merged**